modules/machineset-yaml-osp-sr-iov-port-security.adoc

```diff
@@ -92,8 +92,6 @@ spec:
 ====
 Trunking is enabled for ports that are created by entries in the networks and subnets lists. The name of ports that are created from these lists follow the pattern `<machine_name>-<nameSuffix>`. The `nameSuffix` field is required in port definitions.
```

Comment:
Sould this fragment also mention that trunks can be enabled at the Ports level?

Reply:
So, "You can enable trunking per port," "You can enable trunking per `Port` object/resource," or something else? @MaysaMacedo

Reply:
I guess "you can enable trunking per Port" might sound better

Reply:
```suggestion
Trunking is enabled for ports that are created by entries in the networks and subnets lists. The name of ports that are created from these lists follow the pattern `<machine_name>-<nameSuffix>`. The `nameSuffix` field is required in port definitions.

You can enable trunking for each port.
```

@MaysaMacedo ?

And

Reply:
Also, TODO: match in other mod.